content/scout/policy/_index.md

```diff
@@ -214,3 +201,49 @@ in the CLI.
 > is enabled. Docker Scout doesn't have access to historic evaluations. Trigger
 > a SonarQube analysis and policy evaluation after enabling the integration to
 > view the results in Docker Scout.
+
+### Approved base images
+
+The **Approved base images** policy lets you restrict which base
+images you allow in your builds.
+
+This policy checks whether the base images used in your images match any of the
+reference globs specified in the policy configuration. The following table
+shows a few example globs for this policy.
+
+| Effect                                                          | Glob                                  |
+| --------------------------------------------------------------- | ------------------------------------- |
+| Allow any images from Docker Hub                                | `hub.docker.com/**`                   |
+| Allow Docker Hub images under a specific organization           | `hub.docker.com/dockerscoutpolicy/**` |
+| Allow images on a registry with hostname `registry.example.com` | `registry.example.com/**`             |
+| Allow slim variants of NodeJS                                   | `hub.docker.com/node:*-slim`          |
+
+This policy isn't enabled by default. To enable the policy:
+
+1. Go to the [Docker Scout Dashboard](https://scout.docker.com/).
+2. Go to the **Policies** section.
+3. Select the **Approved base images** policy in the list.
+4. Select **Enable**. The policy configuration page opens.
+5. Enter the glob patterns that you want to allow.
+6. Select **Save and enable**. The policy is now enabled for your current
+   organization.
```

Comment:
**thought:** After 3. Users will land on the configuration page where they can proceed with 5. and 6.
**suggestion:** Omit step 4.
```suggestion
3. Select the **Approved base images** policy in the list.
4. Enter the glob patterns that you want to allow.
5. Select **Save and enable**. The policy is now enabled for your current
   organization.
```

Reply:
Done! thanks!